languages/pt.json

```diff
@@ -2,150 +2,151 @@
-    "FILE_EMPTY": "The file is empty",
-    "FORBIDDEN_ACTION_DIR": "Unable to perform action. Forbidden for directories.",
-    "FORBIDDEN_ACTION_FILE": "Unable to perform action. Forbidden for files.",
+    "FILE_EMPTY": "O arquivo está vazio",
+    "FORBIDDEN_ACTION_DIR": "Não é possível executar ações. Proibido para diretórios.",
+    "FORBIDDEN_ACTION_FILE": "Não é possível executar ações. Proibido para arquivos.",
     "FORBIDDEN_CHAR_SLASH": "O uso de '/' é proibido no diretório ou arquivo nome.",
-    "FORBIDDEN_CHANGE_EXTENSION": "You are not allowed to change file extension.",
-    "FORBIDDEN_NAME": "Forbidden name '%s'.",
+    "FORBIDDEN_CHANGE_EXTENSION": "Você não tem permissão para alterar a extensão do arquivo.",
+    "FORBIDDEN_NAME": "Nome proibido '%s'.",
     "INVALID_ACTION": "Ação inválida.",
     "INVALID_DIRECTORY_OR_FILE": "Pasta ou arquivo inválido.",
     "INVALID_FILE_TYPE": "Tipo de arquivo não aceito.",
-    "INVALID_FILE_UPLOAD": "Upload de arquivo inválido.",
+    "INVALID_FILE_UPLOAD": "Carregar de arquivo inválido.",
     "INVALID_VAR": "Variável %s inválida.",
     "NOT_FOUND_LANGUAGE_FILE": "Arquivo de idioma não encontrado.",
-    "NOT_FOUND_SYSTEM_MODULE": "System module \"%s\" not found.",
-    "MODE_ERROR": "Mode error.",
-    "NOT_ALLOWED": "Você não está autorizado a processar essa acão",
-    "NOT_ALLOWED_SYSTEM": "As permissoões do sistema não permite que você realize esta ação",
-    "STORAGE_SIZE_EXCEED": "Maximum storage size %s has been exceeded.",
-    "UNABLE_TO_CREATE_DIRECTORY": "Não foi possível criar a pasta %s.",
-    "UNABLE_TO_OPEN_DIRECTORY": "Não foi possível abrir  pasta %s.",
+    "NOT_FOUND_SYSTEM_MODULE": "Módulo do sistema '%s' não encontrado.",
+    "MODE_ERROR": "Erro do modo.",
+    "NOT_ALLOWED": "Você não está autorizado a processar essa ação",
+    "NOT_ALLOWED_SYSTEM": "Como permissões do sistema não permitem que você realize essa ação",
+    "STORAGE_SIZE_EXCEED": "O tamanho máximo de armazenamento %s foi excedido.",
+    "UNABLE_TO_CREATE_DIRECTORY": "Não foi concebido para criar uma pasta %s.",
+    "UNABLE_TO_OPEN_DIRECTORY": "Não foi possível abrir pasta %s.",
     "UPLOAD_FILES_SMALLER_THAN": "Favor enviar apenas arquivos menores que %s.",
     "UPLOAD_IMAGES_ONLY": "Favor enviar apenas imagens. Outros arquivos não são aceitos.",
     "UPLOAD_IMAGES_TYPE_JPEG_GIF_PNG": "Favor enviar apenas imagens no formato JPG, GIF ou PNG.",
-    "action_copy": "Copy",
+    "action_copy": "Copiar",
     "action_delete": "Excluir",
-    "action_download": "Download",
-    "action_extract": "Extract",
-    "action_move": "Move",
+    "action_download": "Baixar",
+    "action_extract": "Extrair",
+    "action_move": "Mover",
     "action_rename": "Renomear",
-    "action_replace": "Sobrepor arquivo",
+    "action_replace": "Substituir",
     "action_select": "Selecionar",
     "action_upload": "Enviar",
+    "action_loading": "Solicitação %s de %s em andamento.",
```

Comment:
I added "action_loading" when an action is requested

Reply:
![image](https://user-images.githubusercontent.com/5089350/29888308-dbd9cedc-8d96-11e7-89ed-bc91fedd734f.png)